src/views/approvalsView.js

```diff
@@ -6,12 +6,15 @@
 import React, { useEffect, useState } from 'react';
 import { makeStyles, useTheme } from '@material-ui/core/styles';
 import Grid from '@material-ui/core/Grid';
+import SearchIcon from '@material-ui/icons/Search';
+import Alert from '@material-ui/lab/Alert';
 import { useSelector, useDispatch } from 'react-redux';
-import { Tabs, Tab, Divider, colors } from '@material-ui/core';
+import { Tabs, Tab, Divider, colors, Paper, Input } from '@material-ui/core';
 import ApprovalCard from '../components/requests/approvalsCard';
 import Pagination from '../components/common/Pagination';
 import Header from '../components/common/SettingsHeader';
 import { getManagerRequests } from '../redux/actions/requestApprovalAction';
+import searchAction from '../redux/actions/searchAction';
 
 const drawerWidth = 240;
 export default function ApprovalsTable({ history }) {
```

Comment:
@erickyvand any reason why this PR contains changes in the unrelated files i.e. approvalsView?

Reply:
@bdushimi, It contains changes because that file contains requests and I needed to search in those requests